test/e2e/lib/pages/gutenboarding/acquire-intent-page.js

```diff
@@ -17,7 +17,9 @@ export default class AcquireIntentPage extends AsyncBaseContainer {
 
 	async enterSiteTitle( siteTitle ) {
 		const siteTitleSelector = By.css( '.acquire-intent-text-input__input' );
-		return await driverHelper.setWhenSettable( this.driver, siteTitleSelector, siteTitle );
+		return await driverHelper.setWhenSettable( this.driver, siteTitleSelector, siteTitle, {
+			pauseBetweenKeysMS: 10,
```

Comment:
Adding this pause makes it so that the text is actually entered. Without it there was a race condition that would cause it to fail, which is why it was disabled earlier today.

Reply:
Is the race condition fixed?

Reply:
Thanks @bsessions85 for making the test pass again! :+1:

In #49108 I spent some significant time trying to find the underlying bug in Gutenboarding code itself. But I haven't succeeded. I found and fixed a few suspicious and potentially buggy places, did some performance optimizations, but the e2e test still fails exactly the same way as it did before.

It's entirely possible  there is a bug in the Selenium WebDriver and how it interacts with Chrome.